composer.json

```diff
@@ -46,10 +46,12 @@
         "symfony/yaml": "~2.3",
         "symfony/security-acl": "~2.4",
         "sonata-project/intl-bundle": "~2.1",
-        "symfony/phpunit-bridge": "~2.7|~3.0"
+        "symfony/phpunit-bridge": "~2.7|~3.0",
+        "jms/di-extra-bundle": "dev-master as 1.6"
```

Comment:
Should be `~1.6`.

Maybe a `conflict` key should be added if not working with earlier versions.